IBAnimatable/Utils.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Utils.swift
+//  IBAnimatableApp
+//
+//  Created by jason akakpo on 03/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import Foundation
+
+internal extension Array {
+  /// Returns the element at the specified index iff it is within bounds, otherwise nil.
```

Comment:
iff is here for "If and only if" but if this naming is not enough widespread I can change it.